pytransform3d/plot_utils/_plot_functions.py

```diff
@@ -94,33 +94,33 @@ def plot_box(ax=None, size=np.ones(3), A2B=np.eye(4), ax_s=1, wireframe=True,
 
 def plot_sphere(ax=None, radius=1.0, p=np.zeros(3), ax_s=1, wireframe=True,
                 n_steps=20, alpha=1.0, color="k"):
-    """Plot sphere.
+    """Plot sphere(s).
 
     Parameters
     ----------
     ax : Matplotlib 3d axis, optional (default: None)
         If the axis is None, a new 3d axis will be created
 
     radius : float, optional (default: 1)
-        Radius of the sphere
+        Radius of the sphere(s)
 
-    p : array-like, shape (3,), optional (default: [0, 0, 0])
-        Center of the sphere
+    p : array-like, shape (:,3), optional (default: [0, 0, 0])
```

Comment:
Please define the shape as `(n_spheres, 3)`, so

1. give the number of elements in the first dimension a name
2. PEP8 compatibility (also in the docstring): space after `,`